tests/core/pyspec/eth2spec/test/merge/fork/test_merge_fork_random.py

```diff
@@ -0,0 +1,120 @@
+from random import Random
+
+from eth2spec.test.context import (
+    with_phases,
+    with_custom_state,
+    with_presets,
+    spec_test, with_state,
+    low_balances, misc_balances, large_validator_set,
+)
+from eth2spec.test.utils import with_meta_tags
+from eth2spec.test.helpers.constants import (
+    PHASE0, MERGE,
+    MINIMAL,
+)
+from eth2spec.test.helpers.merge.fork import (
+    MERGE_FORK_TEST_META_TAGS,
+    run_fork_test,
+)
+from eth2spec.test.helpers.random import (
+    randomize_state,
+    randomize_attestation_participation,
+)
+
+
+@with_phases(phases=[PHASE0], other_phases=[MERGE])
+@spec_test
+@with_state
+@with_meta_tags(MERGE_FORK_TEST_META_TAGS)
```

Comment:
We can just define:
```python

def composed(*decs):
    def deco(f):
        for dec in reversed(decs):
            f = dec(f)
        return f
    return deco

merge_test = composed(
  with_phases(phases=[PHASE0], other_phases=[MERGE]),
  spec_test,
  with_state,
  with_meta_tags(MERGE_FORK_TEST_META_TAGS),
)

@merge_test
def test_something_fun(spec, phases, state):
   ...
```

Lots of other places where we can combine decorators as well.